Gemfile

```diff
@@ -2,15 +2,19 @@ source 'https://rubygems.org'
 
 gemspec
 gem 'fog-aws'
-gem 'capybara', '~> 2.3.0'
-gem 'database_cleaner', '~> 0.9.1'
-gem 'rspec-rails', '~> 3.2.0'
-gem 'poltergeist'
 gem 'ammeter'
-gem 'test-unit', platforms: [:ruby_22]
 
-group :development do
+group :development, :test do
```

Comment:
What about development?, the dummy app uses the same Gemfile: https://github.com/withassociates/slices/blob/rails-4/spec/dummy/Gemfile